packages/color-picker/src/draggable.js

```diff
@@ -0,0 +1,36 @@
+import isServer from '@element-plus/utils/isServer'
+let isDragging = false
+
+export default function(element, options) {
+  if (isServer) return
+  const moveFn = function(event) {
+    if (options.drag) {
+      options.drag(event)
+    }
+  }
+  const upFn = function(event) {
+    document.removeEventListener('mousemove', moveFn)
```

Comment:
use `on,off`

Reply:
wait...,i will change draggable.js to ts and resolve this code

Reply:
done